e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/add-payment.spec.js

```diff
@@ -193,4 +198,88 @@ context('PDC_RECONCILE users can add a payment to a report', () => {
     cy.getInputByLabelText('Yes').should('not.be.checked');
     cy.getInputByLabelText('No').should('not.be.checked');
   });
+
+  describe('when user navigates away', () => {
+    const FEE_RECORD_ID_THREE = '33';
+    const FEE_RECORD_ID_FOUR = '44';
+
+    beforeEach(() => {
+      const feeRecordThree = FeeRecordEntityMockBuilder.forReport(report)
+        .withId(FEE_RECORD_ID_THREE)
+        .withPaymentCurrency(PAYMENT_CURRENCY)
+        .withStatus(FEE_RECORD_STATUS.TO_DO)
+        .build();
+      const feeRecordFour = FeeRecordEntityMockBuilder.forReport(report)
+        .withId(FEE_RECORD_ID_FOUR)
+        .withPaymentCurrency(PAYMENT_CURRENCY)
+        .withStatus(FEE_RECORD_STATUS.TO_DO)
+        .build();
+
+      cy.task(NODE_TASKS.REMOVE_ALL_UTILISATION_REPORTS_FROM_DB);
+      cy.task(NODE_TASKS.REMOVE_ALL_PAYMENTS_FROM_DB);
+      cy.task(NODE_TASKS.INSERT_UTILISATION_REPORTS_INTO_DB, [report]);
+
+      const feeRecords = [feeRecordOne, feeRecordTwo, feeRecordThree, feeRecordFour];
+      cy.task(NODE_TASKS.INSERT_FEE_RECORDS_INTO_DB, feeRecords);
+
+      const matchingTfmFacilities = getMatchingTfmFacilitiesForFeeRecords(feeRecords);
+      cy.task(NODE_TASKS.INSERT_TFM_FACILITIES_INTO_DB, matchingTfmFacilities);
+    });
+
+    describe('by clicking the back button', () => {
```

Comment:
Can we add "with XYZ checkboxes checked" please?

Reply:
Just pushed a change for this thanks, please let me know if you're not sure on how it's named (I'm not 100% sure on it but can't think of a better alternative)